mesos_collectd.py

```diff
@@ -101,6 +106,17 @@ def configure_callback(conf, is_master, prefix, cluster, instance, path, host,
     })
 
 
+def get_version_from_api(host, port):
+    version_api_url = "http://" + host + ":" + str(port) + "/version"
+    try:
+        result = json.load(urllib2.urlopen(version_api_url, timeout=10))
+        return result['version']
+    except urllib2.URLError, e:
+        collectd.error('%s plugin: Error connecting to %s - %r' %
+                       (PREFIX, version_api_url, e))
+        return None
```

Comment:
Nit: "None" isn't necessary since that's the default return value for functions. I do realize that line 126 does the same thing, which came from the original plugin.